docs/content/en/highlights/2021-07-23-vrl-match_datadog_query.md

```diff
@@ -0,0 +1,133 @@
+---
+date: "2021-07-23"
+title: "Datadog Search Syntax support"
+description: "Use Datadog Search Syntax in `filter` transforms and VRL"
+authors: ["leebenson"]
+pr_numbers: [7837, 8370],
+release: "0.16.0"
+hide_on_release_notes: false
+badges:
+  type: new feature
+  providers: ["datadog"]
+  domains: ["vrl", "filter transform"]
+---
+
+This release adds support for [Datadog Search Syntax](datadog_seach_syntax), a
+query language based loosely on [Lucene](lucene) and designed for parsing
+Datadog log lines.
+
+This allows you to filter log lines  akin to the experience offered by [Datadog
+Live Tailing](datadog_live_tailing).
+
+There's two ways you can incorporate Search Syntax into your Vector workflow:
+
+## 1. As a `filter` transform condition
+
+Assuming the following `vector.toml`:
+
+```toml
+[api]
+  enabled = true
+
+[sources.gen1]
+  type = "generator"
+  format = "json"
+  batch_interval = 0.2
+
+[transforms.filter]
+  type = "filter"
+  inputs = ["gen1"]
+  condition.type = "datadog_search"
+  condition.source = "*mopper"
+
+[sinks.blackhole]
+  type = "blackhole"
+  inputs = ["gen1"]
+  print_amount = 1000
+```
+
+Running `vector tap` will yield log messages affixed with "mopper" - e.g:
```

Comment:
Not a big deal, but you could use `console` above to avoid the need to `tap` it separately though I do also like calling attention to `vector tap` to raise awareness of it.